### ISSUE TITLE
Heroku deleting free dynos

### ISSUE THREAD

**Issue by ktunkiewicz** [Nasicus/d2-holy-grail#106]
Hi @Nasicus , I just got an email from Heroku about this: https://help.heroku.com/RSBRUH58/removal-of-heroku-free-product-plans-faq
and immediately thought about your app. Is it using a free dyno? if so, are you going to pay Heroku?

Your app is very popular, would be nice not to loose everyone's progress when they delete the dynos :(

**Comment by Nasicus** [comment]
yes I also got this mail a while ago. I would like to continue, but I'm not really happy I have to pay... I could move to another free service I guess, but that would change the URL... not sure yet what I'm going to do

**Comment by ktunkiewicz** [comment]
@Nasicus there's not much time for decision on this :( maybe inform users to export their progress just in case?

**Comment by Nasicus** [comment]
@ktunkiewicz I will not just shut it down, if necessary, I will pay for a few months to notify users, but I hope I'll find something else
